artemis-core-client/src/main/java/org/apache/activemq/artemis/core/message/openmbean/MessageOpenTypeFactory.java

```diff
@@ -123,6 +124,7 @@ protected void init() throws OpenDataException {
    public Map<String, Object> getFields(M m, int valueSizeLimit, int deliveryCount) throws OpenDataException {
       Map<String, Object> rc = new HashMap<>();
       rc.put(CompositeDataConstants.MESSAGE_ID, "" + m.getMessageID());
+      rc.put(CompositeDataConstants.PROTOCOL, m.getClass().getSimpleName());
```

Comment:
we could add a method on Message to return a more user friendly word? 

on Message:
   public String getProtocol();
   
   
   then on core you return "CORE" and AMQP you return "AMQP"
   
   ?

Reply:
I already made that change exactly as you and Michael suggest. that code is in the next commit.

Reply:
maybe I should squash to prevent confusion?